terraform_provider_ibm_z_linuxone/TFE/README.md

```diff
@@ -0,0 +1,117 @@
+    ```bash
+    cd your-local-directory
+    git clone this-repo-url
+    ```
+
+2.  **Push this code to your own git repository**:
+    ```bash
+    git remote add origin <YOUR_REPO_URL>
+    git remote -v
+    git push -u origin main
+    ```
+
+---
+###  Step 2: Configure your Terraform Enterprise
+1. **Create a TFE Workspace**:
+    *   In TFE, create a new workspace linked to your Git repository (VCS connection).
+    *   Choose "Version Control Workflow" and select your Git provider.
+    *   Map the workspace to your repository URL.
+  
+2. **Enable Remote Execution Mode**:
+    *   Select TFE **Remote Execution** (plans and applies run in TFE infrastructure).
+    *   To verify or change this:
+        *   Go to your TFE workspace > **Settings** > **General**.
+        *   Under **Execution Mode**, ensure "Remote" is selected.
+3. **Set Environment Variables in TFE**:
+    *   Create the following environment variables for the *ibm_z_linuxone* provider
+        *   `IBM_Z_LINUXONE_HOST`
+        *   `IBM_Z_LINUXONE_USERNAME` 
+        *   `IBM_Z_LINUXONE_PASSWORD` 
+        *   `IBM_Z_LINUXONE_TLS_INSECURE_SKIP_VERIFY`
+    *   Mark  `IBM_Z_LINUXONE_USERNAME`, `IBM_Z_LINUXONE_PASSWORD`  as sensitive to secure them.
+4. **Edit Version Control Setting**:
+    * Confirm the repository name is correct
+    * Set the *Terraform Working Directory* to match your sample code
+    * Enable *Auto-apply*
+    * Select *Branch-based* for *VCS Trigger Type* and specify the branch name
+    * Set *Automatic Run triggering* to "Always trigger runs"
+
+---
+### Step 3: Pre-Import Preparation
+1. **Find an existing LPAR using HMC**:  
+   Log on to HCM or using HMC APIs to find the existing LPAR you want to update.
+
+2. ** Review code structure**:  
+   The sample code has the following organization for Terraform files:  
+   ```
+   terraform/
+   ├── main.tf          # Provider and resource blocks
+   ├── backend.tf       # TFE backend configuration variables
+   ├── providers.tf     # Terraform and provider versions
+   ``` 
+---
+
+### Step 4: Import an existing LPAR and generate configuration file
+1. **Use the import block**:  
+   Update your [main.tf](terraform/main.tf) to make sure the `import` block matches your environment. 
+
+2. **Configure Backend for TFE**:  
+   Update [backend.tf](terraform/backend.tf) to match your TFE configurations.  
+
+3. **Point to your provider**:  
+   Update the [providers.tf](terraform/providers.tf) to point to the ibm_z_linuxone provider on your private registry
+
+4. **Generate a configuration file**:  
+   Use the following command to generate a configuration file
+   ```bash
+      terraform plan -generate-config-out=generated_resources.tf
+---
+
+### Step 5: Update configuration file and push changes to Github
+1. **Update resource configuration file**:  
+   Update `generated_resources.tf` to increase the CP `initial=` value to a higher number
+
+2. **Commit and push changes to Git**:  
```

Comment:
Users should run terraform plan before committing the change to git to verify the change being made.